modules/setting-up-topology-manager.adoc

```diff
@@ -52,14 +55,17 @@ metadata:
   selfLink: /apis/config.openshift.io/v1/featuregates/cluster
   uid: e802e840-89ee-4137-a7e5-ca15fd2806f8
 spec:
-  featureSet: LatencySensitive
-
-
+  featureSet: LatencySensitive <1>
+...
 ----
+<1> Add the `LatencySensitive` FeatureGate in a comma-separated list.
 
-. Configure the Topology Manager policy with KubeletConfig.
+. Edit the `cpumanager-enabled` custom resource (CR) to configure the Topology Manager policy.
+Create the file if it does not exist. Creating the file also activates CPU Manager.
```

Comment:
@vpickard Am I correct here? The user edits the CPU Manager policy file, if it exists? If not, the user needs to add the parameters to create the CPU manager?

Reply:
Note my question at https://github.com/openshift/openshift-docs/pull/22756/files#r436923741

Reply:
Yes, that's right.